app/components/title-bar/settings-button.tsx

```diff
@@ -12,6 +12,7 @@ export const SettingsButton: FC<BoxProps> = props => {
       mr="tight"
       cursor="default"
       _focus={{ textDecoration: 'underline', outline: 0 }}
+      data-test="btn-settings"
```

Comment:
small nit, i see other places we do `something-btn`, whereas here the `btn-` is the prefix. I don't mind either way, but would be nice to be consistent. Also, would it make sense for all `data-test` attrs to be enum?